src/index.css

```diff
@@ -21,7 +21,7 @@ html {
   --secondary-hover: #feba36;
   --tertiary-hover: #322b2b;
   --success-hover: #30eca8;
-  --danger-hover: #bd0a0a;
+  --danger-hover: #e03e3e;
```

Comment:
is this color change needed? wound't if affect other places using this variable?

Reply:
You're right, I removed it